deploy/olm-catalog/bundle/manifests/konveyor-operator.v99.0.0.clusterserviceversion.yaml

```diff
@@ -202,25 +202,25 @@ spec:
   - konveyor-operator.v1.2.0
   relatedImages:
   - name: controller
-    image: quay.io/konveyor/mig-controller:latest
+    image: quay.io/konveyor/mig-controller:release-1.3.2
   - name: ui
-    image: quay.io/konveyor/mig-ui:latest
+    image: quay.io/konveyor/mig-ui:release-1.3.2
   - name: velero
-    image: quay.io/konveyor/velero:latest
+    image: quay.io/konveyor/velero:konveyor-1.4.2
   - name: velero_restic_restore_helper
-    image: quay.io/konveyor/velero-restic-restore-helper:latest
+    image: quay.io/konveyor/velero-restic-restore-helper:konveyor-1.4.2
```

Comment:
Should these be `konveyor-1.4.3` [(1.4.3)](https://github.com/konveyor/velero/tree/konveyor-1.4.3) to address CVE-2020-3996?